frontend/integration-tests/tests/kubevirt/vm.yaml.scenario.ts

```diff
@@ -1,16 +1,97 @@
 /* eslint-disable no-undef */
+import { execSync } from 'child_process';
 import { browser, ExpectedConditions as until } from 'protractor';
 
 import { appHost, testName } from '../../protractor.conf';
 import { isLoaded } from '../../views/crud.view';
+import * as yamlView from '../../views/yaml.view';
 import { removeLeakedResources } from './utils';
 import { detailViewAction, detailViewVmIcon, runningIcon } from '../../views/kubevirt/vm.actions.view';
+import { disksTab, nicsTab } from '../../views/kubevirt/virtualMachine.view';
 import Yaml from './models/yaml';
-
+import { VirtualMachine } from './models/virtualMachine';
+import { testNAD } from './mocks';
 
 describe('Test create vm from yaml', () => {
-  const leakedResources = new Set<string>();
+  const defaultVM = new Set<string>();
+  const customVM = new Set<string>();
   const yaml = new Yaml();
+  const customVMWithNicDisk = `
```

Comment:
can you move this to mocks? Also dunno if it makes sense to split the mocks file.

Reply:
I got an error below if I put it in mocks, I don't know how to get rid of it so I put it here.
`TypeError: Cannot read property 'split' of undefined`.

Reply:
not sure what you mean here, there shouldn't be any issue with just moving the variable to another file. Where are you reading a 'split' property?

Reply:
https://github.com/kubevirt/web-ui/blob/e3e990178ec8e05f2ef61dcaba02be0b7610818d/frontend/integration-tests/views/yaml.view.ts#L12

Reply:
ah ok

Reply:
But I also don't understand why it doesn't work.

Reply:
It works after rebase on the master now, I will move it to mocks.